Makefile

```diff
@@ -49,6 +53,9 @@ $(GINKGO): Gopkg.toml
 $(OPERATOR_SDK): Gopkg.toml
 	GOBIN=$$(pwd)/build/_output/bin/ go install ./vendor/github.com/operator-framework/operator-sdk/cmd/operator-sdk
 
+$(GITHUB_RELEASE): Gopkg.toml
+	GOBIN=$$(pwd)/build/_output/bin/ go install ./vendor/github.com/aktau/github-release
```

Comment:
Can we pin github-release to a specific release? This doesn't feel safe

Reply:
Done